js/algos.js

```diff
@@ -0,0 +1,135 @@
+// ---------- LONGEST PHRASE
+
+// Accept array of strings as an argument
+// Create 'longest string' variable
+// Loop through array, measuring length of each element
+// If element is longer than the element before it, it gets stored as 'longest string'
+// Return 'longest string'
+
+function longestString(strings){
+  longest = "";
```

Comment:
Remember your best practices in javaScript but using `var` when creating a new variable.